mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -51,27 +52,25 @@ def transform(
         self.__product_type_classifier = wdc_product_type_classifier
 
         # Parse file
-        with open(wdc_jsonl_file_path) as data:
-            for row in data:
-                entry = WdcOffersCorpusEntry.from_json(row)
-                product_type = next(
-                    self.__product_type_classifier.classify(
-                        entry=entry
-                    )
+        for entry in wdc_corpus.entries:
```

Comment:
.entries can just be a generator. Convention is to have generators not be properties.

Something like:
```
def entries(self) -> typing.Generator[WdcCorpusEntry, None, None]:
    with open(self.__jsonl_file_path) as jsonl_file:
        for line in jsonl_file:
            yield WdcCorpusEntry.from_json(json.loads(line))
```